from core.rpc.hola_pb2 import (
    Event as GRPCEvent,
    ReceivableMessageSegment as GRPCReceivableMessageSegment,
    Sender as GRPCSender,
    Reply as GRPCReply,
    Anonymous as GRPCAnonymous,
    Request as GRPCRequest,

    PrivateMessageRequest as GRPCPrivateMessageRequest,
    GroupMessageRequest as GRPCGroupMessageRequest,
    DeleteMsgRequest as GRPCDeleteMsgRequest,
    GetMsgRequest as GRPCRGetMsgRequest,
    GetStrangerInfoRequest as GRPCGetStrangerInfoRequest,
    GetGroupMemberInfoRequest as GRPCGetGroupMemberInfoRequest,
    GetGroupMemberListRequest as GRPCGetGetGroupMemberListRequest,
    SendGroupForwardMsgRequest as GRPCSendGroupForwardMsgRequest,
    SendPrivateForwardMsgRequest as GRPCSendPrivateForwardMsgRequest,

    PrivateMessageResult as GRPCPrivateMessageResult,
    GroupMessageResult as GRPCGroupMessageResult,
    DeleteMsgResult as GRPCDeleteMsgResult,
    GetMsgResult as GRPCGetMsgResult,
    GetMsgResult as GRPCGetMsgResult,
    GetStrangerInfoResult as GRPCGetStrangerInfoResult,
    GetGroupMemberInfoResult as GRPCGetGroupMemberInfoResult,
    GetGroupMemberListResult as GRPCGetGroupMemberListResult,
    SendGroupForwardMsgResult as GRPCSendGroupForwardMsgResult,
    SendPrivateForwardMsgResult as GRPCSendPrivateForwardMsgResult,
)
